packages/react/src/global-alert/GlobalAlert.tsx

```diff
@@ -17,17 +17,20 @@ export function GlobalAlert({ children, onDismiss, title }: GlobalAlertProps) {
 	return (
 		<section
 			aria-label={title}
-			css={{ backgroundColor: boxPalette.systemWarningMuted }}
+			css={{
+				backgroundColor: boxPalette.systemWarningMuted,
+				position: 'relative',
+			}}
 		>
 			<Content>
 				<Flex
```

Comment:
I think `position: relative` should be on this element instead of the top level `section`. This is because this elements parent contains the container padding, so the close button element doesn't need to know about that.